tests/testthat/test-expressions.R

```diff
@@ -320,4 +330,15 @@ with_test_authentication({
         skip("'which' isn't implemented correctly")
         expect_identical(which(duplicated(ds$v4)), 3:20)
     })
+
+    test_that("rollupResolution can be set", {
+        #ensure that only the rollup resolution changes
+        body <- ds$v5@tuple@body
+        expect_null(rollupResolution(ds$v5))
+        rollupResolution(ds$v5) <- "M"
+        expect_identical(rollupResolution(ds$v5), "M")
+        comp <- mapply(identical, body, ds$v5@tuple@body, SIMPLIFY = TRUE)
```

Comment:
What if we ran this as `mapply(expect_equal...` or, even ran it in a for loop?

Reply:
Also does this v5 already have other view settings? Maybe assert what those are before the resolution setting and then also that they are the same after. I think that would be a bit clearer to our future selves.

Reply:
Right, it's not that we're worried that another `body` field is edited. That is covered elsewhere. We were concerned that if body were like

```
list(
    name="something",
    ...,
    view=list(
        some_setting=TRUE
    )
)
```

that `view$some_setting` was still unchanged.